File: cukcuk clone/app/src/main/java/vn/com/misa/cukcukstarterclone/ui/login/LoginContract.java
package vn.com.misa.cukcukstarterclone.ui.login;

import vn.com.misa.cukcukstarterclone.base.BaseContract;

/**
 * @created_by KhanhNQ on 07-Feb-2021.
 */
public class LoginContract {
    interface View extends BaseContract.View {
        void loggedInSuccess();
    }

    interface Presenter extends BaseContract.Presenter<View> {
        void login(String username, String password);
    }
}

File: app/src/main/java/com/misa/fresher/ui/login/LoginContract.kt
package com.misa.fresher.ui.login

import com.misa.fresher.base.BaseContract
import com.misa.fresher.data.model.User

class LoginContract:BaseContract {
    interface View : BaseContract.View{
        fun loginSuccess()
    }
    interface Presenter : BaseContract.Presenter<View>{
        fun login(user : User)
    }
}